bigtable/autoscaler/strategies.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Google Inc.
```

Comment:
Any reason to have this as a separate file? For the sake of samples, it seems it would be most useful to locate this near the code that uses these strategies.